src/ldf-client-ui.js

```diff
@@ -753,16 +753,19 @@ require('leaflet/dist/images/marker-shadow.png');
                 lat = feature.geometry.coordinates[1];
               }
               if (lon && lat) {
-                const marker = L.circleMarker([lat, lon]).addTo(self.map);
+                let marker = L.circleMarker([lat, lon]);
+                self.trialLayerGroup.addLayer(marker).addTo(self.map);
                 if (valueLabel)
                   marker.bindPopup(feature.properties.name).openPopup();
               }
             },
           }).addTo(self.map);
 
-          // Possibly rescale map view
-          self.map.fitBounds(self.mapLayer.getBounds());
+          self.trialLayerGroup.addLayer(laayer).addTo(self.map);
 
+          // Possibly rescale map view
+          // self.map.fitBounds(self.mapLayer.getBounds());
```

Comment:
Unneeded comments?

Reply:
Yes, going to remove them in my next commit